api/agentpb/agent.proto

```diff
@@ -167,6 +167,13 @@ message StartActionRequest {
   // PTSummaryParams describes parameters for PT summary.
   message PTSummaryParams {}
 
+  // PTMgDbSummaryParams describes parameters for PT MongoDB summary.
+  message PTMgDbSummaryParams {
```

Comment:
Command is called pt-mongodb-summary, everywhere in this file we MongoDB, so why should we start using new MgDb here? Next time before creating new variable, please check at least the same file for usages.

Reply:
Sometimes it uses "mongodb", sometimes "mongo". The same for postgres - "posgresSQL", "pg", "pgSQL". I haven't found any clear key to name it. If there was no problem with "pg" or "pgSQL", I don't understand why there is with "MgDb".

Reply:
Could you point where we used pgsql?
`pg` is used for `pg_stat_monitor`, `pg_stat_statements`, `pt-pg-summary` as you can see it's used because of naming in 3rd party tools.
`postgres` used only with `postgres_exporter` for the same reason.
`PostgreSQL` in all other places.

Reply:
I used pgSQL here: https://github.com/percona/pmm/pull/614/files
And it has two approvals, one by yourself.
I can use full words, I don't have problem with that. But I wouldn't like to read then comments like "this is Linux, not Windows".

Reply:
in that case it should be `pg`, not `pgSQL`. So we had 3 options and you decided to add 4th one? Why did you decide to use `pgSQL` there? We missed that and it should be fixed there.

Reply:
Maybe my sense for uniformity: mySQL - pgSQL - mgDQ. Maybe a question for Freud. 
Why? Why not? You have the same deal of guilt as I do since you've approved. Don't try to escape from this...